app/build.gradle

```diff
@@ -4,7 +4,7 @@ android {
     compileSdkVersion 28
     defaultConfig {
         applicationId "io.github.djunicode.salespersontracker"
-        minSdkVersion 19
+        minSdkVersion 24
```

Comment:
If you are getting invoke customs error, you can resolve it without changing min api

Reply:
I had changed the min SDK version to use an app to access the server on my phone , it has nothing to do with the app. The app will work with 19, I'm sorry I didn't remember to change it back